### ISSUE TITLE
Are the Ember + Apollo tutorials deprecated? 

### ISSUE THREAD

**Issue by alexking** [howtographql/howtographql#641]
Noticed that all frontend tutorials other than Apollo + React were removed [in this commit](https://github.com/howtographql/howtographql/commit/e0a3d1eaf73c6f83599c8246c840bbfb45d4d280). I'm guessing this is because they were using the old version of graphcool still? 

- Are there plans to update these tutorials targeting the new version? 
- Are you looking for contributions to update these? (I believe @DevanB was the original writer for the [Ember + Apollo](https://www.howtographql.com/ember-apollo/0-introduction/) one?)

Wanted to bring it up here in case other people were looking for them.

**Comment by DevanB** [comment]
@alexking Yep, the tutorials are definitely out-of-date, including the Ember tutorial! 

I'd gladly welcome another contributor to help me update it! 

If you want to open a PR, work through updating, and ask questions, feel free to @ mention me and I'll give it a look!

**Comment by swcarlosrj** [comment]
@alexking @DevanB thanks for the feedback. By now we're gonna try to fix all the issues & merge PRs, after that we'd discuss how we can improve the platform. I'd let you know in case that'd be possible 😄

**Comment by DevanB** [comment]
Ah, forgot about this issue. I’ve been rewriting the entire tutorial while I’m unemployed and looking for opportunities. I’m writing one section per day, so should have it updated next week!

**Comment by swcarlosrj** [comment]
@DevanB awesome I cannot ensure that it'll be merged (it doesn't depend on me) but it'd definitely help to push forward this platform by & for the community 💜 Let me know If I could help with something!
